shared/charts/couchdb-prometheus-exporter/templates/deployment.yaml

```diff
@@ -29,13 +29,13 @@ spec:
         - containerPort: {{ .Values.exporterListenPort }}
         env:
         - name: COUCHDB_URI
-          value: {{ .Values.couchdbUri }}
+          value: {{ .Values.couchdb.uri }}
         - name: COUCHDB_USERNAME
-          value: {{ .Values.couchdbUsername }}
+          value: {{ .Values.couchdb.username }}
         - name: COUCHDB_PASSWORD
-          value: {{ .Values.couchdbPassword }}
+          value: {{ .Values.couchdb.password }}
```

Comment:
Looks like this should be added to GPII-4074 and also populated via secret reference.

Reply:
I have added this and other affected workloads to the [GPII-4074](https://issues.gpii.net/browse/GPII-4074) ticket description.